### PR TITLE
Vet clinic database: Query & update animals table

### PR DESCRIPTION
**Pull request by Sevenpros**
## In this pull request, we updated the table using different SQL queries as well as answered some analytical questions.
aggregate functions
delete transaction
Transaction with savepoint
update species transaction

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,6 +7,11 @@
```
Comment (brytebee) [inline README.md:7]: - [ ] **Optional:** You do not need to specify `Project Name: VET CLINIC`. Just `VET CLINIC` would serve.

File: queries.sql [hunk 2]
```diff
@@ -9,3 +9,30 @@ select * from animals where neutered = 't';
...
+COMMIT;
+
+/*Working with aggregate functions*/
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
+SELECT species, MIN(weight_kg), MAX(weight_kg) FROM animals GROUP BY species;
+SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-01-01' GROUP BY species;
```
Comment (brytebee) [inline queries.sql]: - [x] **Required:** The date you have specified would begin 1990 jan 1st and end 2000 jan 1st but the remaining months of the year 2000 would be left out. Kindly update it to `2000-12-31` if it matches your format.

**Review by brytebee — CHANGES_REQUESTED** [review]
Hi, @Sevenpros👋,

##

### Highlights💡
- Good commit message.✔️ 
- Good PR Summary.✔️
- Great schema design and great queries.✔️
- App functions as it should.✔️

##

Good job so far coming so far with the progress you've made! 💪💪
 
However, there are some issues that you still need to work on to go to the next project but you are almost there!



# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Please, remember to tag me in your question so I can receive the notification.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @Sevenpros,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- Good pull request message with screenshot ✔️
approval

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**